README.md

```diff
@@ -95,13 +118,9 @@ As you can see, you can mix and match dsc resources with common puppet resources
 All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
 
 ## Limitations
-- DSC Composite resources not yet fully supported.
+- DSC Composite Resources are not yet fully supported.
 - PS Objects like 'PSCredential' as parameters value not yet supported.
 
 ## Notes
-The puppet types are build from the source code of the DSC Resources MOF schema files.
-If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](https://github.com/msutter/puppet-dsc/blob/master/README_BUILD.md)
-
-## License
```

Comment:
It looks like my thoughts are not totally out of the realm of craziness, but it seems we don't need to do much more than put both copyrights on there. 

* http://softwarefreedom.org/resources/2012/ManagingCopyrightInformation.html (see "Maintaining centralized copyright notices")

I think the copyright notice becomes something like:

~~~md
Copyright (c) 2014-Present Marc Sutter, Puppet Labs, original authors
~~~